web/rainmaker/dev-packages/egov-bpa-dev/src/ui-utils/commons.js

```diff
@@ -165,9 +185,38 @@ export const createUpdateBpaApplication = async (state, dispatch, status) => {
     //   })
     // });
 
-    let documents = [
-      {"documentType": "OWNER.IDENTITYPROOF.VOTERID"}
-    ];
+    let documents;
+    let wfDocuments;
+    if(method === 'UPDATE'){
+      documents = payload.documents;
+      //hard coding these values but time being untill we fix documents capture issue.
```

Comment:
Make this `//TODO: ` so it is easier to locate later

Reply:
done.